handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -491,18 +500,29 @@ public final void setTicketKeys(byte[] keys) {
 
     /**
      * Returns the pointer to the {@code SSL_CTX} object for this {@link ReferenceCountedOpenSslContext}.
-     * Be aware that it is freed as soon as the {@link #release()}  method is called.
+     * Be aware that it is freed as soon as the {@link #release()} method is called.
      * At this point {@code 0} will be returned.
+     *
+     * @deprecated Dont use.
```

Comment:
`Done use.` -> `This method is considered unsafe. Other methods on this class should be used and feature requests should be made if capabilities are missing.`